gradle/libs.versions.toml

```diff
@@ -17,9 +17,18 @@ espressoCore = "3.2.0"
 mockk = "1.9.3"
 robolectric = "4.3.1"
 startup = "1.0.0"
+composeActivity = "1.3.1"
+composeMaterial = "1.0.5"
+composeTooling = "1.0.5"
```

Comment:
These will always be the same so renaming this to `compose` and re-using should be fine.

Reply:
Thanks, fixed now.